### ISSUE TITLE
CKAN can't find certain mod files???

### ISSUE THREAD

**Issue by Sanix25** [KSP-CKAN/CKAN#1526]
CKAN is having problems finding certain files. I set everything up correctly and made sure to set the correct location in the KSP command line, but it seems like it's unable to install mods. Because of this, the CKAN status log bar when downloading mods or updating mods fills completely, but never completes. It just sits there at 100% with 0 MiB left. Does anyone know a fix? I'm using Mac OS X Yosemite by the way, with KSP on steam. Every mod that CKAN is having issues with has already been downloaded and is being updated. 

```
11156 [1] ERROR CKAN.URLHandlers (null) - There was an error while registering the URL handler for ckan:// - Could not find a part of the path "/Users/daneh8/.local/share/applications/mimeapps.list".
11438 [1] ERROR CKAN.URLHandlers (null) -   at System.IO.FileStream..ctor (System.String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, Boolean anonymous, FileOptions options) [0x00000] in <filename unknown>:0 
  at System.IO.FileStream..ctor (System.String path, FileMode mode, FileAccess access, FileShare share) [0x00000] in <filename unknown>:0 
  at (wrapper remoting-invoke-with-check) System.IO.FileStream:.ctor (string,System.IO.FileMode,System.IO.FileAccess,System.IO.FileShare)
  at System.IO.StreamWriter..ctor (System.String path, Boolean append, System.Text.Encoding encoding, Int32 bufferSize) [0x00000] in <filename unknown>:0 
  at System.IO.StreamWriter..ctor (System.String path) [0x00000] in <filename unknown>:0 
  at (wrapper remoting-invoke-with-check) System.IO.StreamWriter:.ctor (string)
  at System.IO.File.WriteAllLines (System.String path, System.String[] contents) [0x00000] in <filename unknown>:0 
  at CKAN.URLHandlers.RegisterURLHandler_Linux () [0x00000] in <filename unknown>:0 
  at CKAN.URLHandlers.RegisterURLHandler (CKAN.Configuration config, IUser user) [0x00000] in <filename unknown>:0 
868539 [Threadpool worker] ERROR CKAN.ModuleInstaller (null) - Failure in locating file /Users/daneh8/Library/Application Support/Steam/SteamApps/common/Kerbal Space Program/GameData/FerramAerospaceResearch/FARForceDataUpdate.cfg : Could not find file "/Users/daneh8/Library/Application Support/Steam/SteamApps/common/Kerbal Space Program/GameData/FerramAerospaceResearch/FARForceDataUpdate.cfg"
4808056 [Threadpool worker] ERROR CKAN.ModuleInstaller (null) - Failure in locating file /Users/daneh8/Library/Application Support/Steam/SteamApps/common/Kerbal Space Program/GameData/SCANsat/PluginData : Could not find file "/Users/daneh8/Library/Application Support/Steam/SteamApps/common/Kerbal Space Program/GameData/SCANsat/PluginData"
4815430 [Threadpool worker] ERROR CKAN.ModuleInstaller (null) - Failure in locating file /Users/daneh8/Library/Application Support/Steam/SteamApps/common/Kerbal Space Program/GameData/TestFlight/Properties : Could not find file "/Users/daneh8/Library/Application Support/Steam/SteamApps/common/Kerbal Space Program/GameData/TestFlight/Properties"
```

**Comment by plague006** [comment]
Possibly related to https://github.com/KSP-CKAN/CKAN/issues/1498

**Comment by Olympic1** [comment]
Fixed in #1536
